### PR TITLE
Add a test for collections migration task

### PR DESCRIPTION
**Pull request by hackmastera**
Here's how far I got; test fails as follows:

```
    NoMethodError:
       undefined method `all' for CurationConcerns::Collection:Module
     # ./lib/curation_concerns/data_migration/collections_migration.rb:6:in `run'
     # ./spec/lib/curation_concerns/data_migration/collections_migration_spec.rb:22:in `block (3 levels) in <top (required)>'
```

Could use some guidance @tpendragon @escowles

### CODE REVIEW

**Comment by escowles** [comment]
Maybe a namespacing issue?  Does `::Collection.all.each` work?

**Comment by hackmastera** [comment]
@escowles thanks; that cleared the error!

File: lib/tasks/curation_concerns.rake [hunk 1]
```diff
@@ -10,18 +10,9 @@ namespace :curation_concerns do
...
-          member.save
-        end
-        collection.members = []
-        collection.save
-      end
+      CurationConcerns::DataMigration::CollectionsMigration.run
     end
   end
-end
-namespace :curation_concerns do
```
Comment (escowles) [inline lib/tasks/curation_concerns.rake]: Do we need to leave this in, or won't it change the namespace of the following rake task?

File: spec/lib/curation_concerns/data_migration/collections_migration_spec.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    it 'moves relationship from collection#members to curation_concern#member_of_collections' do
+      expect(collection1.members.to_a.size).to eq 2
+      expect(collection2.members.to_a.size).to eq 2
+      expect(work1.member_of_collections.to_a.size).to eq 0
+      expect(work2.member_of_collections.to_a.size).to eq 0
+      expect(work3.member_of_collections.to_a.size).to eq 0
+      described_class.run
+      expect(collection1.reload.members.to_a.size).to eq 0
+      expect(collection2.reload.members.to_a.size).to eq 0
+      expect(work1.reload.member_of_collections.to_a.size).to eq 1
```
Comment (escowles) [inline spec/lib/curation_concerns/data_migration/collections_migration_spec.rb]: Can you check the ids of the collections here just to make sure the works stay in the right collections?

File: lib/tasks/curation_concerns.rake (hunk 1 above)
Comment (hackmastera) [inline lib/tasks/curation_concerns.rake]: Do we? I'll put it back in; it looked like duplication to me.

File: spec/lib/curation_concerns/data_migration/collections_migration_spec.rb (hunk 2 above)
Comment (hackmastera) [inline spec/lib/curation_concerns/data_migration/collections_migration_spec.rb]: sure thing

File: lib/tasks/curation_concerns.rake (hunk 1 above)
Comment (hackmastera) [inline lib/tasks/curation_concerns.rake]: it's still available at `curation_concerns:solr:reindex`

**Comment by hackmastera** [comment]
No idea what is up with this https://travis-ci.org/projecthydra/curation_concerns/jobs/216900075#L1769

**Comment by escowles** [comment]
I don't know what that error is either, but @tpendragon mentioned it yesterday.

**Comment by hackmastera** [comment]
@tpendragon Can you at least accept this PR into your branch? the failing test exists in both.

**Pull request merged**